src/modules/environment.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line prefer-destructuring
```

Comment:
Kindly do not disable the linters since they are there to help you produce the best standard code whilst conforming to standard practices. Please consider removing this line so that you can work within the confines of the linters.